source/main/index.js

```diff
@@ -206,26 +206,26 @@ const onAppReady = async () => {
 
   buildAppMenus(mainWindow, cardanoNode, locale, {
     isUpdateAvailable: false,
-    isOnInitialSettings: false,
+    isNavigationEnabled: true,
```

Comment:
@yakovkaravelov don't you think we should disable the navigation by default?

Reply:
@nikolaglumac Fixed.